flink-libraries/flink-table/src/main/scala/org/apache/flink/table/sources/CsvTableSource.scala

```diff
@@ -44,15 +47,15 @@ import org.apache.flink.table.api.TableException
   * @param lenient Flag to skip records with parse error instead to fail, false by default.
   */
 class CsvTableSource(
-    path: String,
-    fieldNames: Array[String],
-    fieldTypes: Array[TypeInformation[_]],
-    fieldDelim: String = CsvInputFormat.DEFAULT_FIELD_DELIMITER,
-    rowDelim: String = CsvInputFormat.DEFAULT_LINE_DELIMITER,
-    quoteCharacter: Character = null,
-    ignoreFirstLine: Boolean = false,
-    ignoreComments: String = null,
-    lenient: Boolean = false)
+    private val path: String,
```

Comment:
Actually, I want to access these fields in the `equals(Object)` by `that.path`. The `private val` modifier will create getter/setter implicitly.